portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/logging/CompanyLogging.testcase

```diff
@@ -22,75 +23,341 @@ definition {
+			Navigator.openWithAppendToBaseURL(urlAppend = "o/company-log");
+
+			CompanyLogging.assertCompanyLogIsolated(portalInstanceName = "www.able.com");
 
+			CompanyLogging.assertCompanyLogIsolated(portalInstanceName = "www.baker.com");
+		}
 	}
 
 	@description = "If company.log.enabled=false, there should not be additional logging files created for each company and the UI access point should be unavailable"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CompanyLoggingDisabled {
+		property custom.properties = "company.log.enabled=false";
+
+		task ("When the user tries navigating to the company logging UI access point with company logs disabled") {
+			var baseURL = PropsUtil.get("portal.url");
 
-		// TODO LRQA-75853
+			Open.openNoError(locator1 = "${baseURL}/o/company-log");
+		}
+
+		task ("Then the page cannot be accessed") {
+			AssertElementNotPresent(
+				key_virtualHost = "liferay.com",
+				locator1 = "CompanyLogging#COMPANY_LOG_TITLE");
+
+			AssertLocation.assertPartialLocation(value1 = "web/guest");
+		}
+
+		task ("When the user adds a virtual instance") {
+			HeadlessPortalInstanceAPI.addPortalInstance(
+				domain = "www.able.com",
+				portalInstanceId = "www.able.com",
+				virtualHost = "www.able.com");
+		}
+
+		task ("Then additional company logging files should not be created") {
+			var homeDir = PropsUtil.get("liferay.home.dir.name");
+
+			var companiesFolderExists = FileUtil.exists("${homeDir}/logs/companies");
 
+			if ("${companiesFolderExists}" == "true") {
+				fail("logs/companies folder exists");
+			}
+		}
 	}
 
 	@description = "An Omniadmin can get log names from all companies and download logs from all companies though the UI"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test GetAllCompanyLogs {
+		property test.assert.warning.exceptions = "true";
+		property test.name.skip.portal.instance = "CompanyLogging#GetAllCompanyLogs";
 
-		// TODO LRQA-75854
+		task ("Given all company logs output unique logs") {
+			ServerAdministration.openServerAdmin();
 
+			ServerAdministration.addCategoryLogLevels(
+				categoryLevel = "INFO",
+				categoryName = "com.liferay.portal.background.task.internal.messaging.RemoveOnCompletionBackgroundTaskStatusMessageListener");
+		}
+
+		task ("And given two new virtual instances are created") {
+			HeadlessPortalInstanceAPI.addPortalInstance(
+				domain = "www.able.com",
+				portalInstanceId = "www.able.com",
+				virtualHost = "www.able.com");
+
+			HeadlessPortalInstanceAPI.addPortalInstance(
+				domain = "www.baker.com",
+				portalInstanceId = "www.baker.com",
+				virtualHost = "www.baker.com");
+		}
+
+		task ("When the Omniadmin navigates to the company logging UI access point") {
+			Navigator.openWithAppendToBaseURL(urlAppend = "o/company-log");
+		}
+
+		task ("Then logs files are present for all companies") {
+			for (var companyTitle : list "liferay.com,www.able.com,www.baker.com") {
```

Comment:
`for (var userRoleTitle : list "${userRoleTitles}") {`
Don't hardcode the list in, name the list since we are re-using it, as example above